docs/source/quickstart/setup.mdx

```diff
@@ -151,65 +151,70 @@ iwr 'https://rover.apollo.dev/win/latest' | iex
 
-On your development machine, first create a new project directory for your Node.js gateway. Then inside that directory, run the following to create a `package.json` file:
+On your development machine, first create a new directory for your router project. Then inside that directory, run the following to install the Apollo Router:
 
-```shell
-npm init
+```bash
+curl -sSL https://router.apollo.dev/download/nix/latest | sh
 ```
 
-Next, install the following required libraries:
+This installs the `router` executable in your project directory. You can try running it with the following command:
 
-```shell
-npm install apollo-server @apollo/gateway graphql
+```
+./router
 ```
 
-Finally, create a file named `index.js` and paste the following into it as a minimal (not-yet-functional) gateway implementation:
+If you do, you'll get a startup error message like the following:
 
-```js title="index.js"
-const { ApolloServer } = require('apollo-server');
-const { ApolloGateway } = require('@apollo/gateway');
+<ExpansionPanel title="Click to expand">
 
-const gateway = new ApolloGateway();
+```
+Apollo Router v0.12.0 // (c) Apollo Graph, Inc. // Licensed as ELv2 (https://go.apollo.dev/elv2)
 
-const server = new ApolloServer({
-  gateway,
-  // Subscriptions are not currently supported in Apollo Federation
-  subscriptions: false
-});
+⚠️  The Apollo Router requires a composed supergraph schema at startup. ⚠️
 
-server.listen().then(({ url }) => {
-  console.log(`🚀 Gateway ready at ${url}`);
-}).catch(err => {console.error(err)});
-```
+👉 DO ONE:
 
-This code demonstrates the necessary steps for creating a gateway:
+  * Pass a local schema file with the '--supergraph' option:
 
-1. We initialize an `ApolloGateway` instance.
+      $ ./router --supergraph <file_path>
 
-2. We initialize an `ApolloServer` instance and pass it our gateway via the `gateway` option.
+  * Fetch a registered schema from Apollo Studio by setting
+    these environment variables:
 
-3. We call `listen` on our server instance to begin listening for incoming requests.
+      $ APOLLO_KEY="..." APOLLO_GRAPH_REF="..." ./router
 
-If we run this code as-is with `node index.js`, we get an error:
+      For details, see the Apollo docs:
+      https://www.apollographql.com/docs/router/managed-federation/setup
 
+🔬 TESTING THINGS OUT?
+
+  1. Download an example supergraph schema with Apollo-hosted subgraphs:
+
+    $ curl -L https://supergraph.demo.starstuff.dev/ > starstuff.graphql
+
+  2. Run the Apollo Router with the supergraph schema:
+
+    $ ./router --supergraph starstuff.graphql
 ```
-Error: When a manual configuration is not provided, gateway requires an Apollo configuration.
-```
 
-That's because we aren't currently providing a supergraph schema to our gateway! We'll fix that soon.
+</ExpansionPanel>
+
+That's because we aren't currently providing a supergraph schema to the router! We'll fix that soon.
 
 ## 3. Obtain your subgraph schemas
```

Comment:
While setting this up, it did feel like I was jumping around from tool to tool in every section, making it harder to understand the cohesion between sections. 

So here is my wild suggestion, what if the order of this article was more like this:
1. Install Rover
2. Create a Studio Account
3. Authenticate Rover with Studio
4. Create a router project directory
5. Obtain your subgraph schemas

So the user starts by setting up Rover, then signing up for Apollo Studio, then authenticating Rover, before installing the Router and creating their `.graphql` files. The user directly transitions from setting up their subgraph files to learning about managed federation in the next quickstart step.

Reply:
Great suggestion! I've updated the order and lumped all the tool setup steps under a single high-level step.